### DISCUSSION TITLE
[Feature] You can customize the upload and storage location for uploading photos

### DISCUSSION THREAD

**Discussion by pyccl** [immich-app/immich#12380] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I hope that when uploading photos, you can customize the storage location by uploading them to an external library. On the folder page, open a directory and click the upload button to upload the photos to this directory (on the folder page, the photos will be uploaded to the current folder location by default, and on other pages, they will be uploaded to a custom location that you can choose and have permission for).

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5453
